### DISCUSSION TITLE
build error

### DISCUSSION THREAD

**Discussion by ArcMantis** [MetaCubeX/ClashMetaForAndroid#266] [Q&A]
I am a C++ engineer and I don't know much about Android. Today I tried to compile a v2.10.1 version [2832c9ee4c7efaba2137f59b58cfb26c00715207] arm v8a architecture installation package locally according to README. 
But some errors occurred, it seems that the remote address is inaccessible ->"Could not resolve com.github.kr328.golang:gradle-plugin:1.0.4." This error also appears in the gradle synchronization process in Android Studio. 

The remote address has been deleted and cannot be accessed, so this dependency cannot be installed. So how to solve this problem?

![image](https://github.com/MetaCubeX/ClashMetaForAndroid/assets/58796255/ec952e6e-1e4e-40db-8443-9dc1b4633ee7)

**Comment by dukezhao** [comment]
I met the same error when building，anyone else ？pls.help . I followed the readme to process , when run this step ,  ./gradlew app:assembleMeta-AlphaRelease, the error occured . it's "Could not HEAD 'https://jitpack.io/com/github/kr328/golang/gradle-plugin/1.0.4/gradle-plugin-1.0.4.pom'. Received status code 403 from server: Forbidden"
